infra/modules/identity-provider-client/main.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_cognito_user_pool_client" "client" {
+  name         = var.name
+  user_pool_id = var.cognito_user_pool_id
+
+  callback_urls                = var.callback_urls
+  logout_urls                  = var.logout_urls
+  supported_identity_providers = ["COGNITO"]
+  refresh_token_validity       = 1
+  access_token_validity        = 60
+  id_token_validity            = 60
+  token_validity_units {
+    refresh_token = "days"
+    access_token  = "minutes"
+    id_token      = "minutes"
+  }
```

Comment:
_(non-blocking question)_ I'm assuming you just guessed at these numbers?

Reply:
I brought these values over from a previous project that established these as reasonable defaults. Do you think they should be something else?